therapy/query.py

```diff
@@ -470,7 +470,7 @@ def search_groups(self, query: str) -> Dict:
 
         # check concept ID match
         record = self.db.get_record_by_id(query_str, case_sensitive=False)
-        if record and record['src_name'].lower() not in PROHIBITED_SOURCES:
+        if record and record['src_name'].lower():
```

Comment:
I don't think you need the second condition
```suggestion
        if record:
```